tutorials/combine-error-mitigation-options-with-the-estimator-primitive/combine-error-mitigation-options-with-the-estimator-primitive.ipynb

```diff
@@ -261,16 +261,16 @@
    "source": [
     "### Step 3: Execute circuits using the Estimator primitive\n",
     "\n",
-    "Now, we're ready to run our circuit using the Estimator primitive.\n",
+    "You are now ready to run your circuit using the Estimator primitive.\n",
     "\n",
-    "We'll submit 5 separate jobs, starting with no error suppression or mitigation, and successively enabling various error suppression and mitigation options available in Qiskit Runtime. For information about the options, you can refer to the following pages:\n",
+    "Here you will submit five separate jobs, starting with no error suppression or mitigation, and successively enabling various error suppression and mitigation options available in Qiskit Runtime. For information about the options, refer to the following pages:\n",
     "\n",
     "- [Overview of all options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/options)\n",
     "- [Dynamical decoupling](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.DynamicalDecouplingOptions)\n",
     "- [Resilience, including measurement error mitigation and zero-noise extrapolation (ZNE)](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2)\n",
     "- [Twirling](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions)\n",
     "\n",
-    "Because our jobs can run independently of each other, we'll use [batch mode](https://docs.quantum.ibm.com/run/run-jobs-batch) to allow Qiskit Runtime to optimize the timing of their execution."
+    "Because your jobs can run independently of each other, you can use [batch mode](https://docs.quantum.ibm.com/run/run-jobs-batch) to allow Qiskit Runtime to optimize the timing of their execution."
```

Comment:
```suggestion
    "Because these jobs can run independently of each other, you can use [batch mode](https://docs.quantum.ibm.com/run/run-jobs-batch) to allow Qiskit Runtime to optimize the timing of their execution."
```